wicket-console/src/main/java/ru/ydn/wicket/wicketconsole/Initializer.java

```diff
@@ -2,18 +2,28 @@
 
 import org.apache.wicket.Application;
 import org.apache.wicket.IInitializer;
+import org.apache.wicket.protocol.http.WebApplication;
 
+import com.google.inject.Inject;
+
+import de.agilecoders.wicket.webjars.WicketWebjars;
+import de.agilecoders.wicket.webjars.settings.IWebjarsSettings;
 import ru.ydn.wicket.wicketconsole.devutils.WicketConsoleDebugPanel;
 
 public class Initializer implements IInitializer
 {
 	private static final String DEBUG_BAR_CLASS = "org.apache.wicket.devutils.debugbar.DebugBar";
 //	private static final String REGISTER_CONTRIBUTOR_METHOD = "registerContributor";
+
+	@Inject
+	private IWebjarsSettings webjarSettings;
```

Comment:
I tried different ways (that is why it took so much time). If u need button(look as button) with awesome font, with awesome icon , best and simpliest way - using additional depedencies.